projects/1.16/assets/relics-of-gaming/rogblades/lang/zh_cn.json

```
@@ -0,0 +1,42 @@
+{
+   "itemGroup.rog_mod":"拔刀剑：游戏遗物",
+
+  "rogblades.name": "拔刀剑：游戏遗物",
+  "rogblades.desc": "来自游戏中的武器。",
+
+  "item.slashblade.ex.beastlord": "百兽剑王",
+  "item.slashblade.ex.beastlord.desc": "游戏《尼尔：机械纪元》中的武器。",

```

武器desc要不然遵循原文，要不然参考库内拔刀剑2和exblades的翻译，隐喻出处（前半是采游戏内描述，同时按字面义翻译模组文字，后半隐喻游戏本体）
```suggestion
  "item.slashblade.ex.beastlord.desc": "一把以百兽之王为主题的剑。愿人类荣光永存",
```

原文没有句号的情况下不建议加句号，全文通查并修改谢谢